blue_http.opam

```diff
@@ -11,6 +11,7 @@ homepage: "https://github.com/brendanlong/blue-http"
 doc: "https://brendanlong.github.io/blue-http"
 bug-reports: "https://github.com/brendanlong/blue-http/issues"
 depends: [
+  "core"
```

Comment:
Can you sort this below `bisect_ppx`?

Also doesn't cohttp-async already depend on core?

Reply:
I think it does. I've removed this entry